SplunkRumCrashReporting/SplunkRumCrashReporting/CrashReporting.swift

```diff
@@ -86,7 +122,15 @@ func loadPendingCrashReport(_ data: Data!) throws {
     let span = buildTracer().spanBuilder(spanName: exceptionType ?? "unknown").setStartTime(time: now).setNoParent().startSpan()
     span.setAttribute(key: "component", value: "crash")
     if report.customData != nil {
-        span.setAttribute(key: "crash.rumSessionId", value: String(decoding: report.customData, as: UTF8.self))
+        let customData = NSKeyedUnarchiver.unarchiveObject(with: report.customData) as? [String: String]
```

Comment:
According to the docs this can raise `invalidArgumentException` - but it seems you're seeing it simply return nil when given the old `customData: String` type?